### ISSUE TITLE
[HELM3] Helm3 install --help text contains description of Tiller

### ISSUE THREAD

**Issue by squillace** [helm/helm#6603]
Portion of text returned from `helm3 install --help`:

```
To check the generated manifests of a release without installing the chart,
the '--debug' and '--dry-run' flags can be combined. This will still require a
round-trip to the Tiller server.
```

What I expected:
> Do not expect to hear about Tiller from CLI help on Helm3. 

Output of `helm version`:
version.BuildInfo{Version:"v3.0.0-beta.3", GitCommit:"5cb923eecbe80d1ad76399aee234717c11931d9a", GitTreeState:"clean", GoVersion:"go1.12.9"}

Output of `kubectl version`:
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.1", GitCommit:"d647ddbd755faf07169599a625faf302ffc34458", GitTreeState:"clean", BuildDate:"2019-10-02T17:01:15Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3", GitCommit:"2d3c76f9091b6bec110a5e63777c332469e0cba2", GitTreeState:"clean", BuildDate:"2019-08-19T11:05:50Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}

Cloud Provider/Platform (AKS, GKE, Minikube etc.): AKS

**Comment by karuppiah7890** [comment]
@squillace Would you be willing to raise a PR to remove the references to the non-existent tiller in helm3?

**Comment by rimusz** [comment]
addressed in #6604

**Comment by hickeyma** [comment]
@squillace It is the small things that trip you up in the end! :) No matter how much we rinse, little things remain. Thanks for catching this.

**Comment by karuppiah7890** [comment]
This can be closed as the PR #6604  is merged

**Comment by hickeyma** [comment]
Closing as #6604 is merged. I also did a `find/grep` and we look clean now from Tiller (apart from a few unrelated things).
